.travis.yml

```diff
@@ -21,6 +21,9 @@ env:
     - CONDA_DEPENDENCIES="setuptools sphinx cython numpy pytest-cov"
     - PIP_DEPENDENCIES="coveralls"
     - EVENT_TYPE='push pull_request'
+    - CFLAGS="-m64"
+    - LDFLAGS="-m64"
```

Comment:
Are these flags needed on all platforms? If not, I would favor adding them only to the relevant builds below.

Reply:
I'll double check and adjust accordingly.